Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/server/BaseService.java

```diff
@@ -66,7 +72,7 @@ protected AssertionType getAssertion(WebServiceContext context, AssertionType oA
         if (assertion != null) {
             List<String> relatesToList = extractor.getAsyncRelatesTo(context);
             if (NullChecker.isNotNullish(relatesToList)) {
-                assertion.getRelatesToList().add(extractor.getAsyncRelatesTo(context).get(0));
+                assertion.getRelatesToList().add(relatesToList.get(0));
```

Comment:
if relatesToList is not used anywhere else then can we do 
if (NullChecker.isNotNullish(extractor.getAsyncRelatesTo(context))) {
                assertion.getRelatesToList().add(extractor.getAsyncRelatesTo(context).get(0));
            }